File: Admin.Domain/Entities/Authorization/Transaction.cs
using Admin.Domain.Attributes;

namespace Admin.Domain.Entities.Authorization;

public class Transaction : BaseEntity
{
    [EntityField(IsKey = true)]
    public Guid? SubscriberId { get; set; }
    [EntityField(IsKey = true)]
    public string? ProfileCode { get; set; }
    public string? TransactionCode { get; set; }
    public string? Description { get; set; }
    public string? FlagPermission { get; set; }
    public string? FlagOriginalPermission { get; set; }
    public string? Active { get; set; }
}

File: Admin.UI/ClientApp/src/app/_models/authorization/transaction.ts
import { TransactionActions } from "./transactionActions";

export class Transaction {
    SubscriberId!: number;
    TransactionCode!: string;
    Description!: string;    
    FlagPermission!: string;
    FlagOriginalPermission!: string;
    Active!: string;
    Actions!: TransactionActions[];   
}
